data/translations/es/state-data.json

```diff
@@ -117,121 +117,122 @@
   },
   "gu": {
     "hp_link": "https://gec.guam.gov/",
     "registration_link": "https://gec.guam.gov/register/",
     "more_info_link": "https://gec.guam.gov/",
-    "ip_deadline": "hasta 10 días antes del día de las elecciones",
-    "online_deadline": "hasta 10 días antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida 10 días antes del día de las elecciones",
+    "ip_deadline": "sábado, 15 de octubre de 2022",
+    "online_deadline": "",
+    "mail_received_deadline": "a más tardar debe ser recibida viernes, 28 de octubre de 2022",
     "mail_postmarked_deadline": "",
     "confirm_registration_link": "https://gec.guam.gov/validate/"
   },
   "hi": {
     "hp_link": "https://elections.hawaii.gov/",
     "registration_link": "https://olvr.hawaii.gov/",
     "more_info_link": "https://elections.hawaii.gov/voters/registration/",
-    "ip_deadline": "hasta el mismo día de las elecciones",
-    "online_deadline": "hasta 30 días antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida 30 días antes del día de las elecciones",
-    "mail_postmarked_deadline": "",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
+    "online_deadline": "martes, 08 de noviembre de 2022",
+    "mail_received_deadline": "",
+    "mail_postmarked_deadline": "debe tener el sello del correo lunes, 31 de octubre de 2022",
     "confirm_registration_link": "https://olvr.hawaii.gov/"
   },
   "id": {
     "hp_link": "https://idahovotes.gov/",
     "registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/ApplicationInstructions.aspx",
     "more_info_link": "https://idahovotes.gov/voting/",
-    "ip_deadline": "hasta el mismo día de las elecciones",
-    "online_deadline": "hasta 25 días antes del día de las elecciones",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
+    "online_deadline": "viernes, 14 de octubre de 2022",
     "mail_received_deadline": "",
-    "mail_postmarked_deadline": "debe tener el sello del correo a más tardar 25 días antes del día de las elecciones",
+    "mail_postmarked_deadline": "debe tener el sello del correo viernes, 14 de octubre de 2022",
     "confirm_registration_link": "https://elections.sos.idaho.gov/ElectionLink/ElectionLink/VoterSearch.aspx"
   },
   "il": {
     "hp_link": "https://www.elections.il.gov/",
     "registration_link": "https://spanish-ova.elections.il.gov/",
     "registration_link_translated": "true",
     "more_info_link": "https://www.elections.il.gov/Default.aspx",
-    "ip_deadline": "hasta el mismo día de las elecciones",
-    "online_deadline": "hasta 16 días antes del día de las elecciones",
-    "mail_received_deadline": "a más tardar debe ser recibida 28 días antes del día de las elecciones",
-    "mail_postmarked_deadline": "",
+    "ip_deadline": "martes, 08 de noviembre de 2022",
+    "online_deadline": "martes, 18 de octubre de 2022",
+    "mail_received_deadline": "",
```

Comment:
Updated from received to postmarked. Please check this. Also remove empty deadline properties.

Reply:
Confirmed. Illinois has changed from received to postmarked.